force-app/main/default/classes/AccountHelper.cls

```diff
@@ -26,21 +26,69 @@ Methods:
 Remember:
 Always ensure that your helper methods are bulk-safe to handle multiple records efficiently. 
 */
-public with sharing class AccountHelper {
-    
-    public void setTypeProspect(SObject objects){
+public with sharing class AccountHelper extends TriggerHandler {
```

Comment:
Not sure you need to extend the trigger handler here if you're not using it at all.